lib/babel-pipeline.js

```diff
@@ -44,6 +43,8 @@ function validate(conf) {
 		return {testOptions: defaultOptions};
 	}
 
+	const chalk = require('./chalk').get();
```

Comment:
Why did this need moving?

Reply:
This was left over from some previous debugging sorry